BrowserKit/Sources/SummarizeKit/SummarizeController.swift

```diff
@@ -184,17 +183,20 @@ public class SummarizeController: UIViewController, Themeable, Notifiable, CAAni
 
     override public func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-        let impact = UIImpactFeedbackGenerator(style: .medium)
-        impact.prepare()
-        impact.impactOccurred()
-
-        gradient.startAnimating { [weak self] in
-            guard let self else { return }
-            self.closeButton.alpha = 1.0
-            self.view.layer.insertSublayer(backgroundGradient, at: 0)
-            self.viewModel.onShouldShowTabSnapshot()
```

Comment:
@thatswinnie this is the code we discussed previously, related to the comment I had for this [PR](https://github.com/mozilla-mobile/firefox-ios/pull/27947#discussion_r2278899834).

I removed it and didn't see any issues, but would love to get a second pair of eyes in case I missed something and we can always add this back and revert the changes I made here:
https://github.com/mozilla-mobile/firefox-ios/commit/ac091ae611e7f21aa3257d12c3fd6ac9c5625cde

Reply:
Thanks for testing Winnie! Weird, is that only reproducible with the changes on this branch? What iOS and Xcode build are you using to test? 

There are some issue if we're building with Xcode 16 for iOS 26. I see the you still see the blue close button issue, which should have been fixed in this [PR](https://github.com/mozilla-mobile/firefox-ios/pull/28372). Here's a comment on that - https://mozilla-hub.atlassian.net/browse/FXIOS-13048?focusedCommentId=1122530

This change will also be added to a nightly build `60217` as well because there are some issues we see on main that we don't see on nightly.

cc: @issammani

Reply:
Discussed with Winnie in coffee chat, reverting `ac091ae611e7f21aa3257d12c3fd6ac9c5625cde` and adding back the `self.viewModel.onShouldShowTabSnapshot()` fixes the issue even though it seems not needed for iOS 26. Will revert. thanks @thatswinnie!

Reply:
@thatswinnie I was wondering why we need to know about the content container, when we are showing the summarize view. I think it's because we see it when the view background is clear and we don't have a background color to hide the view behind. Instead of reverting the removal of the broswer content hiding code, I decided to ensure we have a background color, which seems to work. 

Updated in 186b6c9c6b4b4240a98b5e2fba228d759dbc1c18